webcompat/templates/dashboard/triage/needstriage-list.html

```diff
@@ -0,0 +1,156 @@
+            box-shadow: 0 1px 6px rgba(0, 0, 0, 0.2);
+    border-color: rgba(0, 0, 0, 0.2);
+  }
+
+  .wc-Triage.is-hidden {
+    display: none;
+  }
+
+  .wc-Triage-title {
+    font-size: .875em;
+    border-bottom: 1px solid #e9e9e9;
+    padding: 1em;
+    text-overflow: ellipsis;
+    max-width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .wc-Triage-title > * {
+    color: #484848;
+    text-decoration: none;
+  }
+
+  .wc-Triage-title > a:hover {
+    text-decoration: underline;
+  }
+
+  .wc-Triage-infos {
+    padding: 1.2em;
+  }
+
+  .wc-Triage-metas {
+    font-size: .75em;
+  }
+
+  .wc-Triage-metas-infos {
+    font-weight: bold;
+    margin: .4em 0 0 0;
+  }
+
+  .wc-Triage-footer {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between;
+    font-size: .75em;
+    padding: 0 1.2em 1.2em 1.2em;
+  }
+
+  .wc-Triage-link {
+    color: #49a9ee;
+    text-decoration: none;
+  }
+
+  .wc-Triage-link:hover {
+    text-decoration: underline;
+  }
+</style>
+
+<section class="wc-Triages is-hidden" id="js-Triages">
+    {% for issue in needstriage_list %}
+  <article
+    data-number="{{ issue['number'] }}"
+    class="
+      wc-Triage
+      {% if issue['priority'] %} priority {% endif %}
+      {% if issue['needinfo'] %} needinfo {% endif %}
+      {% for browser in issue['browsers'] %}
+        {{ browser }}
+      {% endfor %}
+    "
+  >
+    <div class="wc-Triage-container">
+      <h2 class="wc-Triage-title">
+          <a
+            href="/issues/{{ issue['number'] }}"
+            title="{{ issue['title'] }}"
+          >
+            {{ issue['number'] }}: {{ issue['title'] }}
+          </a>
+        </h2>
+      <div class="wc-Triage-infos">
+        <div class="wc-Triage-metas">
+          <div>
+            Created:
+            <time
+              class="wc-Triage-date"
+              datetime="{{ issue['created_at'] }}"
+            >
+             {{ issue['created_at'] }}
+            </time>
+          </div>
+          <div>
+            Updated:
+            <time
+              class="wc-Triage-date"
+              datetime="{{ issue['updated_at'] }}"
+            >
+              {{ issue['updated_at'] }}
+            </time>
+          </div>
+          <div class="wc-Triage-metas-infos">
+            {% if issue['priority'] %} Priority {% endif %}
+            {% if issue['needinfo'] %} - Needinfo {% endif %}
+          </div>
+        </div>
+      </div>
+      <div class="wc-Triage-footer">
+        <div>
+          {% for browser in issue['browsers'] %}
+            <span>{{ browser }} </span>
+          {% endfor %}
+        </div>
+        <div>
+          <a
+            class="wc-Triage-link"
+            href="/issues/{{ issue['number'] }}"
+            title="{{ issue['title'] }}"
+          >
+            More informations
```

Comment:
ah. @magsout I had missed that. Sorry. 

More information. Without s.
There's no plural for information.